test/fixtures/test-runner/bailout/parallel-loading/infinite-loop.mjs

```diff
@@ -0,0 +1,10 @@
+import { describe, it } from 'node:test';
+
+describe('infinite loop test', () => {
+  it('infinite loop that should be cancelled', async () => {
+    // This should be cancelled by the bail
+    while (true) {
+      await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
Same comment about `timers/promises`. I'll stop pointing it out, but it exists in some other places in this PR.

Reply:
done